README.md

```diff
@@ -33,12 +33,58 @@ git clone https://github.com/stjude-dnb-binfcore/sc-rna-seq-snap.git
 cd /home/rstudio/sc-rna-seq-snap/analyses/<module_of_interest>
 ```
 
+## To run the code in this repository wiht your own data:
+
+1. Fork the repository on your own account from the main page of the `stjude-dnb-binfcore/sc-rna-seq-snap` by clicking the “Fork” button
+
+<img width="650" alt="how-to-fork-repo-1" src="https://github.com/user-attachments/assets/1fc0a459-2c8c-4d2e-ab6b-6abaafae963e">
+
+
+
+
+2. Change the name if you like, but probably not; click “Create fork”
+
+<img width="650" alt="how-to-fork-repo-2" src="https://github.com/user-attachments/assets/914a3db5-6e87-41fb-baf2-a50ffdb2a7c0">
+
+
+3. Enjoy your new project repo!
+
+<img width="650" alt="how-to-fork-repo-3" src="https://github.com/user-attachments/assets/073abb78-3993-4527-a574-859fd3046d39">
+
+
+4. Replace the `project_parameters.Config.yaml` with your file paths and parameters.
+
+5. How to run a module analysis
+
+Before running a module, you will do `synk fork` of your project repo at GitHub, if your branch is behind the main branch of the `stjude-dnb-binfcore/sc-rna-seq-snap:main`. This will update the main branch of your project repo with the new code and modules (if any). This will add code and not break any analyses already run in your project repo. 
```

Comment:
```suggestion
Before running a module, you will do `sync fork` of your project repo at GitHub, if your branch is behind the main branch of the `stjude-dnb-binfcore/sc-rna-seq-snap:main`. This will update the main branch of your project repo with the new code and modules (if any). This will add code and not break any analyses already run in your project repo. 
```